content/en/observability_pipelines/quickstart/_index.md

```diff
@@ -0,0 +1,184 @@
+{{% /tab %}}
+{{< /tabs >}}
+
+### Buffering
+Observability Pipelines includes multiple buffering strategies that allow you to increase the resilience of your cluster to downstream faults. The provided sample configurations use disk buffers, the capacities of which are rated for approximately 10 minutes of data at 10Mbps/core for Observability Pipelines deployments. That is often enough time for transient issues to resolve themselves, or for incident responders to decide what needs to be done with the observability data.
+
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+For AWS, Datadog recommends using the `io2` EBS drive family. Alternatively, the `gp3` drives can be used as well.
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+For Azure AKS, Datadog recommends using the `default` (also known as `managed-csi`) disks.
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+For Google GKE, Datadog recommends using the `premium-rwo` drive class, because that is backed by SSDs. The HDD-backed class, `standard-rwo`, likely does not provide enough write performance for the buffers to be useful.
+{{% /tab %}}
+{{< /tabs >}}
+
+## Connect the Agent and the Worker
+To send logs and metrics to the Observability Pipelines Worker, update your agent configuration with the following:
+
+```yaml
+vector:
+  logs.enabled: true
+  # Adjust protocol to https if TLS/SSL is enabled on the Observability Pipelines Worker
+  logs.url: "http://<OPW_HOST>:8282"
+  metrics.enabled: true
+  # Adjust protocol to https if TLS/SSL is enabled on the Observability Pipelines Worker
+  metrics.url: "http://<OPW_HOST>:8282"
+
+```
+
+`OPW_HOST` is the `EXTERNAL-IP` of the load balancer you set up earlier. You can retrieve it by running the following command:
+
+```
+$ kubectl get svc opw-observability-pipelines-worker
+```
+
+At this point, your observability data should be flowing through the OP Worker, and is available for any data processing you want to do. The next section goes through what processing is included by default, as well as additional options that are available.
+
+## Working with data
+The Helm chart provided above have example processing steps that demonstrate Observability Pipelines tools and ensure that data sent to Datadog is in the correct format. 
+
+### Processing logs
+The provided logs pipeline does the following:
+
+- **Tag logs coming through the Observability Pipelines Worker.** This helps determine what traffic still needs to be shifted over to the Worker as you update your clusters. These tags also show you how logs are being routed through the load balancer, in case there are imbalances.
+- **Correct the status of logs coming through the Worker.** Due to how the Datadog Agent collects logs from containers, the provided `.status` attribute does not properly reflect the actual level of the message. We remove this to prevent issues with parsing rules in the backend, where logs are received from the Worker.
```

Comment:
```suggestion
- **Correct the status of logs coming through the Worker.** Due to how the Datadog Agent collects logs from containers, the provided `.status` attribute does not properly reflect the actual level of the message. It is removed to prevent issues with parsing rules in the backend, where logs are received from the Worker.
```